consensus-types/blocks/proto.go

```diff
@@ -60,10 +76,10 @@ func (b *SignedBeaconBlock) Proto() (proto.Message, error) {
 	}
 }
 
-// Proto returns the underlying protobuf beacon block.
+// Proto converts the beacon block to a protobuf object.
 func (b *BeaconBlock) Proto() (proto.Message, error) {
 	if b == nil {
-		return nil, errNilBlock
+		return nil, nil
 	}
```

Comment:
Any reason for this change? I think i saw a comment about it relating to fuzz testing, but I see this change only some structs and not others.

Reply:
See https://github.com/prysmaticlabs/prysm/pull/10885#discussion_r899084704